reproduce.sh

```diff
@@ -17,13 +17,27 @@
 CLUSTERFUZZ_CONFIG_DIR=~/.config/clusterfuzz
 ROOT_DIRECTORY=$(dirname $(readlink -f "$0"))
 
+# Process command line arguments.
```

Comment:
shouldn't --emulator be used here  instead of --install-android-emulator, otherwise it means we provide both arguments when running reproduce.sh. --emulator could mean that we send --install-android-emulator

Reply:
emulator-install-flag might be a confusing name. I'll change it to something like "additional_deps_args". I believe the logic here is what we want, so I'm not sure I understand the concern completely.